batch/batch/driver/scheduler.py

```diff
@@ -188,9 +188,20 @@ def allocate_cores(user, mark):
 
                 if record['cancel']:
                     log.info(f'cancelling job {id}')
-                    await mark_job_complete(self.app, batch_id, job_id, None, None,
-                                            'Cancelled', None, None, None, 'cancelled')
                     should_wait = False
+
+                    async def with_error_handling(id, f):
+                        try:
+                            await f()
+                        except Exception:  # pylint: disable=broad-except
+                            log.info(f'error while cancelling job {id}', exc_info=True)
```

Comment:
This should be log.exception

Reply:
I don't think we should use an ERROR log level if we're ignoring the error and don't care, it seems like it's not actually a problem. Do I misunderstand the severity of this?

Reply:
oh, I understand cotton's comment now. Ignore, sorry.